modules/deploy/pages/deployment-option/cloud/cloud-overview.adoc

```diff
@@ -198,37 +198,40 @@ BYOC with customer-managed VPC::
 --
 =====
 
-== Cloud architecture
+== Redpanda Cloud architecture
```

Comment:
This header reads awkwardly to me. Is it the product "Redpanda Cloud" architecture? Is it the Redpanda "Cloud architecture"? I may be over-analyzing it but I feel there's a better way to phrase this to get the product name in there.

Reply:
@greg-szrama: Yes, the product name is Redpanda Cloud.